components/header/index.styl

```diff
@@ -214,10 +216,29 @@ header
       transition opacity 0.25s
       &:hover
         opacity 0.5
-@media (max-height 701px)
+
+@media (max-height 761px)
   .header-chevron-link-wrapper
     display none
 
   .header-section-nav
     padding-top 15px
 
+@media (min-width 1600px)
+  .header-slide:before
+    width 1200px
+    left "calc(50% - %s)" % 600px
```

Comment:
We have a [`calc()` mixin](https://github.com/artsy/force/blob/5b82a822f08851c1aaa642ef5cea372d2125e635/components/stylus_lib/index.styl#L47), so I believe you can do something like
```stylus
width calc('50% - 600px')
```